src_assets/windows/misc/vdd/smoke-test-vdd-device-helper.ps1

```
@@ -36,6 +36,48 @@ if (-not (Test-Path -LiteralPath $HelperScript)) {
+$originalGetVddDevices = ${function:Get-VddDevices}
+$originalTestVddControlInterfaceAvailable = ${function:Test-VddControlInterfaceAvailable}
+try {
+    $script:installNefconCalls = 0
+    function Invoke-Nefcon([string] $Path, [string[]] $Arguments) {
+        $script:installNefconCalls++
+        if ($Arguments[0] -eq '--install-driver') {
+            return 3010
+        }
+        return 0
+    }
+    function Get-VddDevices {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-VddDevices' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-VddDevices'

(PSUseSingularNouns)